packages/arb-avm-cpp/data_storage/include/data_storage/aggregator.hpp

```diff
@@ -51,6 +51,10 @@ class AggregatorStore {
     void updateLogsProcessedCount(const uint256_t& count);
     void saveMessageBatch(const uint256_t& batchNum, const uint64_t& logIndex);
     std::optional<uint64_t> getMessageBatch(const uint256_t& batchNum);
+    std::unique_ptr<ReadOnlyTransaction> makeReadOnlyTransaction();
+    [[nodiscard]] std::unique_ptr<const ReadOnlyTransaction>
+    makeConstReadOnlyTransaction() const;
```

Comment:
Is there a need for const readonly as opposed to regular readonly?

Reply:
required to use const readonly if want to create transaction from const function.